r120/inref/zindcacheoverflow.m

```diff
@@ -0,0 +1,20 @@
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;								;
+; Copyright (c) 2017 Finxact, LLC. and/or its subsidiaries.     ;
+; All rights reserved.                                          ;
+;								;
+;	This source code contains the intellectual property	;
+;	of its copyright holder(s), and is made available	;
+;	under a license.  If you do not know the terms of	;
+;	the license, please stop and do not read further.	;
+;								;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;
+; Test to check that the reporting of indirect code cache performance statistics does not encounter a 4-byte overflow error
+;
+; Methodology:
+;   Run 43 million set operations, then check that the hit ratio is 99%. If it is zero %, the cache hits counter might have a numeric overflow bug
```

Comment:
1) Why 43 million? Would be good to add a comment 43 million * 100 is slightly over 4Gi and hence that choice for this test.
2) Also, it is "indirection" operations (not SET operations which are only 22 million).
3) And "might have" -> "used to have".
4) "cache hits counter" -> "Hit Ratio"